lib/fontist/cli.rb

```diff
@@ -64,9 +64,26 @@ def update
       STATUS_SUCCESS
     end
 
-    desc "locations MANIFEST", "Get locations of fonts from MANIFEST (yaml)"
-    def locations(manifest)
-      paths = Fontist::Locations.call(manifest)
+    desc "manifest-locations MANIFEST",
+         "Get locations of fonts from MANIFEST (yaml)"
+    def manifest_locations(manifest)
+      paths = Fontist::Manifest::Locations.call(manifest)
+      print_yaml(paths)
+      success
+    rescue Fontist::Errors::ManifestCouldNotBeFoundError
+      error("Manifest could not be found.")
+    rescue Fontist::Errors::ManifestCouldNotBeReadError
+      error("Manifest could not be read.")
+    end
+
+    desc "manifest-install MANIFEST", "Install fonts from MANIFEST (yaml)"
+    option :confirm_license, banner: "yes", desc: "Confirm license agreement"
```

Comment:
Will it be ok for us to have such an option? Could be used as:

```
fontist manifest-install --confirm-license=yes manifest.yml
```

Reply:
Certainly, that works. Or can we do `fontist manifest-install --confirm-license manifest.yml`?

Reply:
Ping @alexeymorozov

Reply:
Yeap, on it :) Thanks